crates/aptos/src/node/mod.rs

```diff
@@ -288,6 +287,16 @@ impl RegisterValidatorCandidate {
     }
 }
 
+#[derive(Parser)]
+/// Register the current account as a Validator candidate
```

Comment:
I'd flip these, keep the derive next to the function because the doc comments can be many lines.